Bots/DotNet/FunctionalTestsBots.sln

```diff
@@ -35,8 +35,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SimpleHostBotComposer", "Co
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EchoSkillBotComposer", "Skills\Composer\EchoSkillBotComposer\EchoSkillBotComposer.csproj", "{9BA47CF9-7D90-4B5C-A9FA-01797A435D53}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ComposerHostBotDotNet", "Consumers\Composer\ComposerHostBotDotNet\ComposerHostBotDotNet.csproj", "{E3FCDDD4-553D-44BA-8B2F-82BA778261BC}"
+EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ComposerSkillBotDotNet", "Skills\Composer\ComposerSkillBotDotNet\ComposerSkillBotDotNet\ComposerSkillBotDotNet.csproj", "{925E0E3F-194D-4E56-AE78-6FF6D49B7E7E}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WaterfallHostBotComposer", "Consumers\Composer\WaterfallHostBotComposer\WaterfallHostBotComposer.csproj", "{80AF5608-8126-439F-B463-197E6A17D941}"
+EndProject

```

Comment:
WaterfallComposer doesn't exist anymore, it is the same bot as this one, remove it from the solution or else it fails at the start, after removing make sure it starts the solution without errors.
![imagen](https://user-images.githubusercontent.com/62260472/151369697-49502b63-2ab3-4079-ab14-c507156f6991.png)
![imagen](https://user-images.githubusercontent.com/62260472/151369859-4d32ac1e-4581-4f37-ac82-35c899fc3969.png)